go-controller/pkg/util/util.go

```diff
@@ -111,6 +111,18 @@ func GetLegacyK8sMgmtIntfName(nodeName string) string {
 	return GetK8sMgmtIntfName(nodeName)
 }
 
+// GetNetworkScopedK8sMgmtHostIntfName returns the management port host interface name for a network id
+// NOTE: network id is used instead of name so we don't reach the linux device name limit of 15 chars
+func GetNetworkScopedK8sMgmtHostIntfName(networkID uint) string {
```

Comment:
can we have simpler prefix like `mp-<networkID>` ? this would save few more characters for vrf device as it would add `-vrf` as suffix for its device name.

Reply:
hehe I guess given primary network has ovn-k8s-mp0 we planned to do ovn-k8s-mp1, ovn-k8s-mp2 etc if you add vrf then oopsy good point :) @pperiyasamy can you while generating the vrf device name trim it to mp1-vrf and mp2-vrf?

Reply:
renaming to mp-<networkID>

Reply:
no no please don't